build_latest.sh

```diff
@@ -157,6 +159,13 @@ function check_build_needed() {
 
 	adopt_image_tag="${tag// -t /}"
 
+	if [ "${runtype}" == "test" ]; then
```

Comment:
Please add a comment explaining about the `runtype` flag

Reply:
```
# `runtype` flag specifies if the script is being run for `build` or `test` (PR checks)
# Checking for runtype if its `test` we proceed to build release images, as part of PR checks
```

Added the following comment which explains about runtype flag